opentripplanner-geocoder/src/main/java/org/opentripplanner/geocoder/pelias/PeliasGeocoder.java

```diff
@@ -0,0 +1,167 @@
+import java.util.List;
+
+import javax.ws.rs.core.UriBuilder;
+
+import org.opentripplanner.geocoder.Geocoder;
+import org.opentripplanner.geocoder.GeocoderResult;
+import org.opentripplanner.geocoder.GeocoderResults;
+
+import com.vividsolutions.jts.geom.Envelope;
+import edu.usf.cutr.pelias.SearchRequest;
+import edu.usf.cutr.pelias.SearchResponse;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.geojson.Feature;
+import org.geojson.Point;
+
+public class PeliasGeocoder implements Geocoder {
+    private String apiKey;
+    private Integer resultLimit;
+    private String viewBox;
+    private String focusPoint;
+    
+    public PeliasGeocoder() {
+    }
+    
+    public String getapiKey() {
+        return apiKey;
+    }
+    
+    public void setapiKey(String Url) {
+        this.apiKey = Url; 
+    }
+
+    public String getFocusPoint() {
+        return focusPoint;
+    }
+    
+    public void setFocusPoint(String str) {
+        this.focusPoint = str; 
+    }
+    
+    public Integer getResultLimit() {
+        return resultLimit;
+    }
+
+    public void setResultLimit(Integer resultLimit) {
+        this.resultLimit = resultLimit;
+    }
+
+    public String getViewBox() {
+        return viewBox;
+    }
+
+    public void setViewBox(String viewBox) {
+        this.viewBox = viewBox;
+    }   
+    
+    @Override 
+    public GeocoderResults geocode(String address, Envelope bbox) {
+        SearchResponse response = null;
+        
+        try {
+
+            Double focusLat, focusLon;
+            if (bbox != null) {
+                focusLat = (bbox.getMinX() + bbox.getMaxX()) / 2;
+                focusLon = (bbox.getMinY() + bbox.getMaxY()) / 2;
+            } else {     
+                focusLat = new Double(focusPoint.split(",")[0]);
+                focusLon = new Double(focusPoint.split(",")[1]);
+            }
+
+            String rectMinLat, rectMinLon, rectMaxLat, rectMaxLon;
+            
+            if (bbox != null) {
+                rectMinLat = Double.toString(bbox.getMinX());
+                rectMinLon = Double.toString(bbox.getMinY());
+                rectMaxLat = Double.toString(bbox.getMinX() + bbox.getWidth());
+                rectMaxLon = Double.toString(bbox.getMinY() + bbox.getHeight());                
+            }
+            else {
+                rectMinLat = viewBox.split(";")[0].split(",")[0];
+                rectMinLon = viewBox.split(";")[0].split(",")[1];
+                rectMaxLat = viewBox.split(";")[1].split(",")[0];
+                rectMaxLon = viewBox.split(";")[1].split(",")[1];        
+            }
+            
+            response = new SearchRequest.Builder(apiKey, address)
+                    .setSources("osm")
+                    .setSize(resultLimit)
+                    .setFocusPoint(focusLat, focusLon)
+                    .setBoundaryRect(rectMinLat, rectMinLon, rectMaxLat, rectMaxLon)                    
```

Comment:
In v1.0.0, the `setBoundaryRect()` method now takes the `Double` datatype, instead of a string.  So you no longer need to do the `Double.toString()` operations above.